xc7/utils/prjxray_edge_library.py

```diff
@@ -1801,9 +1814,13 @@ def create_and_insert_edges(
             if 'PADOUT0' in pip.name or 'PADOUT1' in pip.name:
                 continue
 
-            if pip.is_pseudo:
+            if pip.is_pseudo and "CLK_HROW_CK" not in pip.name:
                 continue
 
+            dless = None
+            if "CLK_HROW_CK" in pip.name:
+                dless = delayless_switch_pkey
```

Comment:
This is the wrong switch to use, why not use the switch associated with the psuedo pip, which had timing through the feature?

Reply:
All right

Reply:
Done